ReactWindows/ReactNative.Shared/Bridge/ReactInstance.cs

```diff
@@ -217,11 +217,14 @@ private string BuildModulesConfig()
             }
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid", Justification = "Reviewed.")]
         private void HandleException(Exception ex)
```

Comment:
Do you need this SuppressMessage here?  It seems that the warning is for the delegate below.